src/test/java/com/microsoft/sqlserver/jdbc/AETest.java

```diff
@@ -0,0 +1,499 @@
+            -5, -26, 107, 79, -48, -1, 71, 104, 22, 115, 66, -94, 73, 88, -48, 71, 44, 118, 15, -76, -98, -74, -78, 43,
+            33, 120, -74, -87, 78, 31, 36, 96, -104, -98, -28, 121, -128, -8, -77, -16, -74, -94, -101, -57, 99, -78,
+            70, 108, -26, 115, 21, -13, -41, 46, -93, -68, 29, 76, 68, 109, -63, 108, 27, -112, -20, -53, 92, 109, 121,
+            33, 64, 69, 42, -72, -45, 35, 50, 60, -124, -99, -40, -7, -96, -57, -14, -71, -103, -44, 13, 69, -89, -17,
+            -46, -128, -5, -83, 80, -125, 13, -48, -126, 49, 62, 37, 113, -92, -30, 15, 47, 113, 124, 108, -89, -48, -4,
+            -84, -61, -11, -55, -97, -28, 36, 20, -106, 57, -16, 107, 51, 67, -95, -73, -76, -8, -118, 97, 127, -20, 67,
+            103, 25, 3, -67, 41, 73, 122, 92, -9, 22, 69, -91, -107, 52, -23, -119, -56, 71, -76, 77, -51, 12, -72, 117,
+            -48, 108, -100, -77, -103, 114, 45, 100, 100, -11, 55, 124, -121, -102, -105, 22, 4, -33, 76, 68, 116, -96,
+            -80, 43, 5, -119, -113, 44, 66, -128, -104, 60, -78, -123, 56, -36, -99, -63, 71, 25, -15, 17, -8, -71, -8,
+            -37, 92, 70, 127, 33, 69, -34, 11, -9, 38, 127, -46, -10, 112, 126, -7, -77, 16, -32, -42, 118, -110, 98,
+            -112, 78, -95, 56, -45, -46, -108, -54, 116, 99, 107, 78, -92, -60, 72, -45, -40, 33, -93, -21, -90, -108,
+            -19, 89, -42, 29, -123, 35, -114, -95, -37, 2, -61, 12, 84, -100, -100, 112, -96, 80, 12, 90, 78, 36, 40,
+            -56, -90, -60, -77, -77, -56, 53, -93, -10, -22, -47, 15, 5, -90, -67, 111, -82, -94, 96, -68, -79, -3, -51,
+            -108, 41, 112, -89, 22, -92, 88, 37, 66, 20, 93, 111, 102, -69, -20, -47, -43, -24, 82, -41, 12, -58, 53,
+            68, -76, -94, -116, 75, 14, 24, -43, 73, -78, -87, 21};
+
+    /**
+     * Setup environment for test.
+     * 
+     * @throws Exception
+     *         when an error occurs
+     */
+    @BeforeAll
+    public static void setUp() throws Exception {
+        connectionStringAE = TestUtils.addOrOverrideProperty(connectionString, "columnEncryptionSetting",
+                ColumnEncryptionSetting.Enabled.toString());
+
+        String enclaveAttestationUrl = TestUtils.getConfiguredProperty("enclaveAttestationUrl");
+        connectionStringAE = TestUtils.addOrOverrideProperty(connectionStringAE, "enclaveAttestationUrl",
+                (null != enclaveAttestationUrl) ? enclaveAttestationUrl : "http://blah");
+
+        String enclaveAttestationProtocol = TestUtils.getConfiguredProperty("enclaveAttestationProtocol");
+        connectionStringAE = TestUtils.addOrOverrideProperty(connectionStringAE, "enclaveAttestationProtocol",
+                (null != enclaveAttestationProtocol) ? enclaveAttestationProtocol : AttestationProtocol.HGS.toString());
+
+        try (SQLServerConnection con = PrepUtil.getConnection(connectionStringAE)) {
+            isAEv2 = TestUtils.isAEv2(con);

```

Comment:
Please verify the exception here. Use TestUtils.formatErrorMsg() to check `R_enclaveNotSupported`.